lib/dvl/components/flashes.rb

```diff
@@ -4,7 +4,7 @@ class Dvl::Components::Flashes < Erector::Widget
   needs :flash
 
   def content
-    @flash.select { |k, v| k.to_s.in?(%w(success error info)) && v.present? }.each do |k, v|
+    @flash.select { |k, v| k.to_s.in?(%w(success error info alert notice)) && v.present? }.each do |k, v|
```

Comment:
Gotcha, just copying code from [this comment](https://github.com/plataformatec/devise/issues/2282#issuecomment-13636842) as it seems to work fine. Probably not worth spending lots of time messing around with Devise for this, right?